tests/Controller/LeagueControllerTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
Writing of the functional tests - it's a good design, but in this case you need to test positive results, as well.  Also in case of covering by this type of tests you need to configure separate database for tests.